contrib/syslog-ng-debun

```diff
@@ -672,20 +678,34 @@ acquire_syslog_pids () {
 	[ -n "$service_status" ] && $service_status >${tmpdir}/svc.pre
 }
 
-acquire_syslog_info () {
-	ls -la "${binprefix}" > ${tmpdir}/syslog.lsl.install_dir 2>&1
-	printf "Syslog-ng's exact version: "
-	$syslogbin --version > ${tmpdir}/syslog.version
-	head -1 ${tmpdir}/syslog.version
-	${syslogbin}-ctl stats > ${tmpdir}/syslog.stats 2>&1
+acquire_syslog_stats () {
+	local tsdata
+	tsdata=$( timestamp )
+	#handy-dandy delay magic, triggered when we'd step on our own foot
+	[ -f ${tmpdir}/syslog.stats.${tsdata} ] && sleep 5 && tsdata=$( timestamp )
+
+	${syslogbin}-ctl stats > ${tmpdir}/syslog.stats.${tsdata} 2>&1
 	echo ${syslogbin}-ctl stats
-	${syslogbin}-ctl query get "*" > ${tmpdir}/syslog.query.all 2>/dev/null
+	tsdata=$( timestamp )
+	${syslogbin}-ctl query get "*" > ${tmpdir}/syslog.query.all.${tsdata} 2>/dev/null
```

Comment:
Maybe we could save the `stderr` also for the query.

Reply:
Saving the stderr output is not really helpful in this case. I don't want the user to see it, and we won't use it do determine why it did not work. The other parts of the script will give us enough information about syslog's state, so that we won't need stderr here.

Reply:
Ok.